doc/common/glossary.md

```diff
@@ -116,48 +158,60 @@ Terms not defined by Kubernetes:
+**Core (physical):** An independent computer processing unit that can independently execute CPU instructions and is integrated with other cores on a multiprocessor (chip, integrated circuit die). Please note that the multiprocessor chip is also referred to as a CPU that is placed in a socket of a computer motherboard.
 
-**Flavor Capabilities** such as (CPU Pinning, NUMA, huge pages): CNTT NFVI Profile
+**Flavor Capabilities** such as (CPU Pinning, NUMA, huge pages): CNTT NFVI Profile.
 
-**Flavor Geometry** Flavor sizing such as number of vCPUs, RAM, disk, etc.
+**Flavor Geometry:** Flavor sizing such as number of vCPUs, RAM, disk, etc.
 
-**[Fluentd](https://www.fluentd.org/)**, a CNCF graduated project, is an open-source data collector for unified logging layer, which allows data collection and consumption for better use and understanding of data.
+**[Fluentd](https://www.fluentd.org/):** An open-source data collector for unified logging layer, which allows data collection and consumption for better use and understanding of data. **Fluentd** is a CNCF graduated project,
 
-**Host profile** is a set of underlay configuration values used to install a new compute host/server. Typically, the host profile configurations match the capabilities of one or more flavors.
+**Host profile:** A set of underlay configuration values used to install a new compute host/server. Typically, the host profile configurations match the capabilities of one or more flavors.
 
 **Hugepages:** Physical memory is partitioned and accessed using the basic page unit (in Linux default size of 4 KB). Hugepages, typically 2 MB and 1GB size, allows large amounts of memory to be utilised with reduced overhead. In an NFV environment, huge pages are critical to support large memory pool allocation for data packet buffers. This results in fewer Translation Lookaside Buffers (TLB) lookups, which reduces the virtual to physical pages address translations. Without huge pages enabled high TLB miss rates would occur thereby degrading performance.
 
-**Kibana** is an open source data visualisation system
+**Kibana:** An open source data visualisation system.
 
-**Prometheus** is an open-source monitoring and alerting system.
+**Prometheus:** An open-source monitoring and alerting system.
 
-**Software Defined Storage (SDS)** architecture consists of the storage software that is independent from the underlying storage hardware. The storage access software provides data request interfaces (APIs) and the SDS controller software provides storage access services and networking.
+**Software Defined Storage (SDS):** An architecture which consists of the storage software that is independent from the underlying storage hardware. The storage access software provides data request interfaces (APIs) and the SDS controller software provides storage access services and networking.
```

Comment:
this is common to OpenStack and Kubernetes and so should this be moved under "Software Layer Terminology"?

Reply:
thanks, done